src/python/py_tokenizer.cpp

```diff
@@ -54,6 +54,7 @@ void init_tokenizer(py::module_& m) {
                 if (max_length.has_value()) {
                     tokenization_params[ov::genai::max_length.name()] = *max_length;
                 }
+                py::gil_scoped_release rel;
```

Comment:
BTW, tokenization and detokenization steps are very fast.. maybe we can avoid playing with GIL here?

Reply:
Agreed, removed.